homeassistant/components/twinkly/light.py

```diff
@@ -210,7 +208,6 @@ async def async_turn_on(self, **kwargs: Any) -> None:
                     self._client.default_mode = "color"
 
         if ATTR_EFFECT in kwargs:
-            _LOGGER.debug("Setting effect '%s'", kwargs[ATTR_EFFECT])
```

Comment:
It's ok to have side effects in entity methods.

Do you still want to remove this?

Reply:
I don't think it is needed, unless you/we/someone wants to keep it until we are sure this works for different light strings